site/_posts/2020-06-03-user-registration-and-sign-in-with-laravel.md

```diff
@@ -311,28 +361,28 @@ Now you can create a `profile.blade.php` file in your `resources/views/` directo
 ```
 {% endraw %}
 
-Any other user data you choose to store in FusionAuth will be available in the `session()->get('user')` array if you'd like to show it to logged in users.
+Any other user data you choose to store in FusionAuth will be available in the `session()->get('user')` array if you'd like to show it to logged-in users.
```

Comment:
I think we use `logged in` (no dash). A search doesn't show any `logged-in` terms. Can you please add that to the devdocs?

Reply:
Good ole Grammarly suggested that `-`.  Will update devDocs 👍🏻